src/libs/actions/Report.js

```diff
@@ -1636,7 +1636,7 @@ function hasAccountIDEmojiReacted(accountID, users, skinTone) {
  * @param {Number} [skinTone]
  */
 function addEmojiReaction(reportID, reportActionID, emoji, skinTone = preferredSkinTone) {
-    const createdAt = moment().utc().format(CONST.DATE.SQL_DATE_TIME);
+    const createdAt = timezoneFormat(utcToZonedTime(new Date(), 'UTC'), CONST.DATE.FNS_DB_FORMAT_STRING);
```

Comment:
Seems like you're transforming a UTC to UTC time?

Maybe see if `UTCDate` from `@date-fns/utc` helps

Reply:
This is a way how to convert to UTC with date-fns-tz, date-fns itself can't convert to utc timezone the date.

Reply:
Ok 👍🏻